ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -219,6 +219,24 @@ objects:
         trap 'touch /tmp/shared/exit' EXIT
         trap 'kill $(jobs -p); exit 0' TERM
 
+        function fips_check() {
+          oc --insecure-skip-tls-verify --request-timeout=60s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodes
+          while read p; do
+            oc debug node/"$p" -- cat /proc/sys/crypto/fips_enabled > /tmp/enabled
```

Comment:
Again, it would be simpler to use a variable:
```bash
            actual_fips="$(oc debug node/"$node" -- cat /proc/sys/crypto/fips_enabled)"
            if [[ "${ENABLE_FIPS}" = true ]] && [[ "${actual_fips}" = 0 ]]; then
￼              # ...
```

Reply:
I started w/ variables, but only way I fully have it functioning is w/ the tmpfiles.  i'll keep tinkering w/ it though